packages/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake

```diff
@@ -26,45 +26,47 @@ set(REACT_COMMON_DIR ${REACT_ANDROID_DIR}/../ReactCommon)
 # If you have ccache installed, we're going to honor it.
 find_program(CCACHE_FOUND ccache)
 if(CCACHE_FOUND)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
```

Comment:
can you revert this change? A good practice is to try and minimize the changes between the two files and this formatting is not necessary.
Plus it will probably requires us to reformatting because of different internal formatting setting.